src/Pos/Wallet/Web/BListener.hs

```diff
@@ -50,6 +50,27 @@ instance ( MonadRealDB m
     onApplyBlocks = onApplyTracking
     onRollbackBlocks = onRollbackTracking
 
+walletGuard
+    :: ( AccountMode m
+       , WithLogger m
+       )
+    => HeaderHash
+    -> CId Wal
+    -> m ()
+    -> m ()
+walletGuard curTip wAddr action = WS.getWalletSyncTip wAddr >>= \case
+    Nothing ->
+        logWarning $
+        sformat ("Wallet Tracking: there is no syncTip corresponding to wallet #"%build) wAddr
```

Comment:
btw, this message is printed on start for `myRootAddress` (or something like that).
there are some wallet-addresses without `walletSyncTip` it was unexpected for me